docs/source/composer_model.rst

```diff
@@ -0,0 +1,248 @@
+|:pickup_truck:| ComposerModel
+==============================
+
+Your Pytorch model and training step must be re-organized as a
+:class:`.ComposerModel` in order to use our :class:`.Trainer`.
+This interface helps our trainer access the necessary parts of your model
+to easily speed up training.
+
+Minimal Example
+---------------
+
+To create a trainable torchvision ResNet-18 classifier with a cross-entropy loss,
+define the |forward| and |loss| methods.
+
+Notice how the forward pass is still under user control (no magic here!)
+and encapsulated together clearly with the architecture.
+
+As an added benefits we take care of:
+
+-  ``x.to(device), y.to(device)``
```

Comment:
Can we use english to describe these functions (maybe keep the code examples in an e.g.).